invenio_communities/views/api.py

```diff
@@ -24,10 +24,16 @@ def init(state):
     ext = app.extensions["invenio-communities"]
 
     # services
-    rr_ext.registry.register(ext.service)
+    rr_ext.registry.register(ext.service, service_id="communities")
```

Comment:
very minor: would it make sense to do `ext.service.id` (see [here](https://github.com/inveniosoftware/invenio-records-resources/blob/master/invenio_records_resources/services/base/service.py#L76)). Although this is something we have not done in any other registration (maybe this is the point when we start fixing it hehe)